ci/check.sh

```diff
@@ -36,11 +36,11 @@ function run() {
 }
 
 if [ ! -f ".venv/bin/activate" ]; then
-    banner "Virtual environment not present in .venv.\nPlease use \`uv venv\` to create one."
+    banner "Virtual environment not present in .venv.\nPlease use \`uv venv --seed\` to create one."
```

Comment:
nit: do we want to specify the python version you should use like in `CONTRIBUTING.md`?
```suggestion
    banner "Virtual environment not present in .venv.\nPlease use \`uv venv --python 3.12 --seed\` to create one."
```